### ISSUE TITLE
Links are not clickable in pdf documents

### ISSUE THREAD

**Issue by Koc** [CollaboraOnline/online#10765]
### Describe the Bug
It is not possible to click on link in pdf documents. But it works fine in docx/odt.

### Steps to Reproduce
1. Upload docx, odt and pdf documents with links
2. Open each document and try to click on link

### Expected Behavior
Link is clickable in every document

### Actual Behavior
Link is clickable in docx and odt but not clickable in pdf

### Screenshots
<details><summary>:mag: Preview</summary>
<p>

![docx](https://github.com/user-attachments/assets/a1acf7b1-b171-471d-8691-453d7f266194)
![odt](https://github.com/user-attachments/assets/af492a6a-625c-467d-9c08-e01782c28e95)
![pdf](https://github.com/user-attachments/assets/f50c844a-60bb-4e57-b402-ef938dd42ad0)

</p>
</details> 

### Desktop
(Please complete the following information)
 - **Collabora version:** 24.04.8-2
 - **OS and version:** Ubuntu 24.04
 - **Browser and version:** Firefox 133

**Comment by gokaysatir** [comment]
I think we don't have this feature yet.

**Comment by Koc** [comment]
@gokaysatir I thought that it's already implemented according to [release notes](https://www.collaboraonline.com/collabora-online-23-05-release-notes/):

> Possibility to follow hyperlinks in read-only documents. Previously links in read-only documents were not clickable. Now they are.

**Comment by gokaysatir** [comment]
> @gokaysatir I thought that it's already implemented according to [release notes](https://www.collaboraonline.com/collabora-online-23-05-release-notes/):
> 
> > Possibility to follow hyperlinks in read-only documents. Previously links in read-only documents were not clickable. Now they are.

Yes we have this for read-only documents. But PDF files fall into a different category.

We added comment features to PDF files but I think we are not getting hyperlink information from core side for PDF files.

**Comment by infocillasas** [comment]
Hi,
I can confirm that internal links are not clickable in PDF documents (Nextcloud v 31.0.7, Collabora CODE v.25.04.4.1 5aa2ead294)
It would be nice also to show the table of contents based on the internal links, like in the PDF viewer App

<img width="999" height="457" alt="Image" src="https://github.com/user-attachments/assets/0957ca05-d84a-43d0-a409-870a06656e93" />

Can someone give the feedback if this features would be added in the next releases ?

**Comment by timur-g** [comment]
Resolved with https://gerrit.libreoffice.org/c/core/+/187926

**Comment by infocillasas** [comment]
> Resolved with https://gerrit.libreoffice.org/c/core/+/187926

Thanks. Will it be available in the next version ?

**Comment by timur-g** [comment]
Yes, in 25.04.5.1 or maybe 25.04.4.2 already available as docker.

**Comment by infocillasas** [comment]
Hi,
I just tested with Collabora Online Development Edition 25.04.4.3 but doesn't work...

**Comment by timur-g** [comment]
Can you pleaes check your version and paste here from Help-About?  CODE has no  25.04.4.3.
The way it works is not direct link open on click, but opening pop-up dialog like with other documents.

<img width="556" height="266" alt="Image" src="https://github.com/user-attachments/assets/d9169126-4aff-4f95-9a67-c1743012c222" />

**Comment by infocillasas** [comment]
<img width="514" height="220" alt="Image" src="https://github.com/user-attachments/assets/c8fed6f3-8ce3-4713-9c30-86db08997587" />
